pyproject.toml

```diff
@@ -61,9 +61,12 @@ flake8 = "^3.9.0"
 flake8-bugbear = "^21.3.2"
 flake8-builtins = "^1.5.3"
 flake8-comprehensions = "^3.4.0"
-flake8-pytest-style = "^1.4.0"
-flake8-logging-format = "^0.6.0"
-flake8-rst-docstrings = "^0.0.14"
+flake8-variables-names = "^0.0.4"
+flake8-simplify = "^0.14.0"
+flake8-fixme = "^1.1.1"
+flake8-annotations = "^2.6.1"
+pytest-sugar = "^0.9.4"
+pytest-icdiff = "^0.5"
```

Comment:
do you really want to add these many dependencies?

Reply:
They are all dev, and I have been removing them slowly. as you can tell by the minuses.

Reply:
I leave them in for this pull. I'm mostly using them to fix up the current code base, and less as permanent fixtures.